### PR TITLE
Add notes about dependencies and commands to install

### PR DESCRIPTION
**Pull request by mppf**

### CODE REVIEW

**Comment by mppf** [comment]
@thomasvandoren how does this look?

File: doc/release/README.prereqs [hunk 1]
```diff
@@ -24,3 +24,15 @@ about your environment for using Chapel:
...
+
+
+We have used the following commands to set up newly installed system images.
+Note that our test system works much better with tcsh installed and that users
+of Chapel won't need 'git'.:
+
+CentOS 6.5, 6.6, 7.0, SLES 11 SP3:
+sudo yum install git gcc gcc-c++ bc perl python tcsh bash
+
+Debigan 7.8, Ubuntu 12.04, 14.04, 14.10:
```
Comment (thomasvandoren) [inline doc/release/README.prereqs]: Debian?

Comment (thomasvandoren) [inline doc/release/README.prereqs]: Did you try using one of the ubuntu cloud images? I think other packages will be needed for that.

https://cloud-images.ubuntu.com/releases/14.10/release/

File: doc/release/README.prereqs [hunk 2]
```diff
@@ -24,3 +24,15 @@ about your environment for using Chapel:
...
   C++ compiler should work, but the aforementioned are the ones we have
   the most experience with.  
+
+
+We have used the following commands to set up newly installed system images.
+Note that our test system works much better with tcsh installed and that users
+of Chapel won't need 'git'.:
+
+CentOS 6.5, 6.6, 7.0, SLES 11 SP3:
+sudo yum install git gcc gcc-c++ bc perl python tcsh bash
```
Comment (thomasvandoren) [inline doc/release/README.prereqs]: I don't think git should be included. Developers may need other things like flex, bison, gcc 4.7+, or python 2.7 depending on what they are working on. It seems unfortunate to complicate the user requirements with the developer requirements (which could go in a doc under doc/developer/).

File: doc/release/README.prereqs [hunk 3]
```diff
@@ -24,3 +24,18 @@ about your environment for using Chapel:
   code using compilers by Cray, IBM, Intel, and PGI. Any standard C and
   C++ compiler should work, but the aforementioned are the ones we have
   the most experience with.  
+
+
+We have used the following commands to set up newly installed system images.
+Note that our test system works much better with tcsh installed and that users
+of Chapel won't need 'git'.:
```
Comment (thomasvandoren) [inline doc/release/README.prereqs]: I don't think git should be included here. I think developer requirements should be kept separate as they are more complicated (e.g. a developer probably needs/wants gcc 4.7+, python 2.7, flex, bison, gdb, valgrind, etc). Maybe they can be listed somewhere in doc/developer/ ?

File: doc/release/README.prereqs [hunk 4]
```diff
@@ -24,3 +24,18 @@ about your environment for using Chapel:
...
+
+CentOS 6.5, 6.6, 7.0:
+sudo yum install git gcc gcc-c++ bc perl python tcsh bash make
+
+SLES 11 SP3:
+sudo zypper install git gcc gcc-c++ bc perl python tcsh bash make
+
+Debian 7.8, Ubuntu 12.04, 14.04, 14.10:
```
Comment (thomasvandoren) [inline doc/release/README.prereqs]: Did you try this with the ubuntu cloud image? I think other things will be needed there:

https://cloud-images.ubuntu.com/releases/14.10/release/

**Comment by thomasvandoren** [comment]
This looks good!  I think it will make it easier for users to get the tools they need installed.

File: doc/release/README.prereqs (hunk 4 above)
Comment (mppf) [inline doc/release/README.prereqs]: I'm working on trying an ubuntu cloud-image.

**Comment by mppf** [comment]
The ubuntu cloud image I tried already had g++ - tcsh was the only thing I ended up installing... but I couldn't get git to work on it (other problems). I think these cloud images will work with the command I have.

**Comment by thomasvandoren** [comment]
Ok, cool. Thanks for looking into that.

**Comment by bradcray** [comment]
I think if we're going to more proactively ask people to read README.prereqs (which is fine with me), then I think we should take a cleanup pass to make it more readable/less verbose (it's bothered me for awhile in that respect, and I'm the one who wrote it).  I'm happy to help with that effort.

**Pull request merged**